### PR TITLE
test: make test fail faster

### PR DESCRIPTION
**Pull request by Lightning00Blade**
Currently if we encounter the WPT fixture that does `fetch` we wait for 12 sec before it fails, which is absurd amount of time.  
Extracting from other PR to see if anything fails.

### CODE REVIEW

**Review by OrKoN — APPROVED** [review]

**Comment by Lightning00Blade** [comment]
@jrandolf Do you have a idea why the test for invalid may be timing out?

**Comment by jrandolf** [comment]
Blocked by https://github.com/GoogleChromeLabs/chromium-bidi/pull/1895

**Comment by Lightning00Blade** [comment]
@jrandolf Still failing after the PR was merged.

**Review by sadym-chromium — APPROVED** [review]
This change is risky, and can cause WPT CI flakiness, so please take an eye on it.

**Comment by Lightning00Blade** [comment]
I have run the CI with this seething multiple times on https://github.com/GoogleChromeLabs/chromium-bidi/pull/1871 only this test ever failed. So I am confident in this change.

**Pull request merged**
